lib/ipython_autoimport.py

```diff
@@ -216,14 +231,34 @@ def _uninstall_namespace(ipython):
 
 def load_ipython_extension(ipython):
     _install_namespace(ipython)
-    # Tab-completion occurs in a different thread from evaluation and history
```

Comment:
I think this is still needed?

Reply:
I don't think so, the call to `_get_import_cache` wasn't changing any global variables, and imports from completion work fine for me.

Reply:
Indeed, that looks fine now.